resources/local/chainspec.toml.in

```diff
@@ -82,7 +82,7 @@ block_gas_limit = 10_000_000_000_000
 
 [wasm_config]
 # Amount of free memory (in 64kB pages) each contract can use for stack.
-initial_memory = 64
+max_memory = 64
```

Comment:
Is this a reasonable max amount?  I don't know it isn't - I'm just asking :)

(Same applies to production chainspec)

Reply:
Personally I think 4MB is too small for reasons explained in the code snippet in referenced JIRA ticket. Not all allocators might play nice with such small limit. I'm not sure how much other smart contract engines give, but at least 2x amount or even more seems reasonable. I'll bring this topic up. Anyway good thing is that now its upgradable